pkg/state/impl/etcd/etcd_test.go

```diff
@@ -57,6 +59,35 @@ func TestPreserveCreated(t *testing.T) {
 	})
 }
 
+func TestDestroy(t *testing.T) {
+	t.Parallel()
+
+	res := conformance.NewPathResource("default", "/")
+
+	withEtcd(t, func(s state.State) {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		var eg errgroup.Group
+
+		err := s.Create(ctx, res)
```

Comment:
I would probably move this to COSI conformance, as it should pass for all state implementations.

Reply:
I can add it there too. And keep it here

Reply:
This repo should run all conformance from COSI anyways.

Reply:
ahh, I see.